main.py

```diff
@@ -24,16 +35,22 @@
     print(cel_to_fahren(args.celsius))
 else:
     answer = str(input("What temperature you want convert?: ")).lower()
-
+    
     while answer not in (CELSIUM_INPUT + FAH_INPUT):
         answer = str(input("What temperature you want convert?: ")).lower()
         if answer in QUIT_INPUT:
             break
-
+    
     if answer in CELSIUM_INPUT:  # Convert from Celsius to Fahrenheit
-        celsius = float(input("Provide temperature in Celsius: "))
-        print(cel_to_fahren(celsius))
-
+        celsius = input("Provide temperature in Celsius: ")
+        if check_number(celsius):
```

Comment:
@MariaMaximova, I think you recreated a bicycle here: there are already methods that can check if the input string is a number or not:
```suggestion
        if celsius.isnumeric():
```

Reply:
@OleksiiBulba No. Please check https://www.w3schools.com/python/ref_string_isnumeric.asp